
author		Francois Mazard <f-mazard@ti.com> 
		Mon, 18 Apr 2011 08:18:48 +0000 (10:18 +0200) 
committer	Francois Mazard <f-mazard@ti.com> 
		Mon, 18 Apr 2011 08:46:52 +0000 (10:46 +0200) 
commit		166d01b32c95b7674b23af3aec82664a3d51002b 
tree		b8fb26894d0d967fc2ba73a152a6232fb0509202 tree 
parent		16bf9a36bba6d7635c035c384fc0b3802339739e commit | diff 

ASoC: ABE DAI: support 2 channels for MODEM DAI

MODEM DAI needs to support 2 channels for dual-mic on capture
and to set the McBSP interface as I2S.

Change-Id: I8996e10219ed40451d955589a3d585d2de345e45
Signed-off-by: Francois Mazard <f-mazard@ti.com>

Index: kernel/sound/soc/omap/omap-abe.c
===================================================================
--- kernel.orig/sound/soc/omap/omap-abe.c	2011-04-18 15:44:24.137105471 +0900
+++ kernel/sound/soc/omap/omap-abe.c	2011-04-18 19:07:12.169647830 +0900
@@ -1671,14 +1671,14 @@
 		.playback = {
 			.stream_name = "Voice Playback",
 			.channels_min = 1,
-			.channels_max = 1,
+			.channels_max = 2,
 			.rates = SNDRV_PCM_RATE_8000 | SNDRV_PCM_RATE_16000,
 			.formats = OMAP_ABE_FORMATS | SNDRV_PCM_FMTBIT_S16_LE,
 		},
 		.capture = {
 			.stream_name = "Voice Capture",
 			.channels_min = 1,
-			.channels_max = 1,
+			.channels_max = 2,
 			.rates = SNDRV_PCM_RATE_8000 | SNDRV_PCM_RATE_16000,
 			.formats = OMAP_ABE_FORMATS | SNDRV_PCM_FMTBIT_S16_LE,
 		},
